src/Components/Select/SelectModal.js

```diff
@@ -413,3 +462,34 @@ SelectModal.propTypes = {
 
 export const SelectModalWithTrans =
   withTranslation("select")(SelectModal);
+
+function getProductsThatMatchArticleTag() {
```

Comment:
should this become an SDK function? just asking, not sure if that should happen

Reply:
That's a good idea, will do it

Reply:
Done